src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmation.js

```diff
@@ -57,6 +57,18 @@ const getFilingPackageData = (
     });
 };
 
+const checkRejectedFiles = (files, setShowRejectedMsg) => {
+  let hasRejectedDoc = false;
+  if (files && files.length > 0) {
+    files.forEach((file) => {
+      if (file.actionDocument && file.actionDocument.status === "REJ") {
+        hasRejectedDoc = true;
+      }
+    });
+  }
+  setShowRejectedMsg(hasRejectedDoc);
```

Comment:
You could exit this loop early as soon as you find your first rejected document. It would be slightly faster overall
```suggestion
    files.forEach((file) => {
      if (file.actionDocument && file.actionDocument.status === "REJ") {
        setShowRejectedMsg(true);
        return;
      }
    });
  }
  setShowRejectedMsg(false);
```

Reply:
Actually, this might not work so easily with a forEach loop. I think a return statement here might just return from the callback function back into the loop. Hmm.

Reply:
Yeah, good thought.  I did do that originally, but it didn't short-circuit like I was planning and always returned false (last statement).

Reply:
Yeah. You could switch this to a for loop I suppose, but I think this is fine as is too.